### PR TITLE
Modular Contracting Site Overhaul

### PR DESCRIPTION
**Pull request by mheadd**
This PR includes a number of changes to the content and structure of the modular contracting site.

* Restructures content into strategy pages for agile, open source, devops, etc.
* Adds examples of projects that we have worked on.
* Uses updated Federalist template and new images for page headers.
* Fixes issue with how [menus render on mobile devices](https://github.com/18F/Modular-Contracting-And-Agile-Development/issues/21).

You can [preview changes here](https://federalist-proxy.app.cloud.gov/preview/18f/modular-contracting-and-agile-development/site-overhaul/).

### CODE REVIEW

**Comment by mheadd** [comment]
@jposi Just a quick note to point out that the [example page](https://github.com/18F/Modular-Contracting-And-Agile-Development/blob/site-overhaul/_projects/vendor-pools.md) that had previously mentioned the State of Ohio has been modified to talk about our work in Mississippi on their agile vendor pool. There is a link at the bottom of the page to the Ohio case, but it's no longer mentioned in the text on the page.

**Comment by awfrancisco** [comment]
This may be a template issue and not specific to this site, but the responsive design breaks for the interior pages (like the toolkit) break to tablet size really early. On my laptop, if the window is even an inch less than full screen, the nav jumps to the top of the page, which looks a little strange when you're expecting a desktop layout/experience. Though this might be better addressed in another issue.

File: _data/navbar.yml [hunk 1]
```diff
@@ -1,39 +1,79 @@
...
+            -
+                text: 'Streamlining Procurement'
+                href: pages/projects/vendor-pools.md
+                show_in_menu: false
+                show_in_footer: false
     -
-        text: 'Using COTS'
-        href: pages/using-cots.md
-        permalink: /using-cots/
+        text: 'Assets'
```
Comment (awfrancisco) [inline _data/navbar.yml:63]: I'd suggest rethinking this term. Unless it's common in program office and contracting circle, it seems like tech/GitHub jargon that we're bringing over into an informational site on another topic.

File: _projects/ak-medicaid.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: project
+title: Helping Those in Need
+description: Using Agile methodologies and DevOps to overhaul the State of Alaska's Medicaid system.
+permalink: /projects/ak-medicaid/
+tags: project
+image: /assets/img/project-images/alaska.jpg
```
Comment (awfrancisco) [inline _projects/ak-medicaid.md:7]: Add caption? People in a field doesn't obviously connect to modernizing a technology system.

File: _projects/ca-child-welfare.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+layout: project
+title: Protecting At Risk Kids
+description: Using modular contracting to help the State of California upgrade its legacy child welfare system.
+permalink: /projects/ca-child-welfare/
+tags: project
+image: /assets/img/project-images/california.jpg
```
Comment (awfrancisco) [inline _projects/ca-child-welfare.md:7]: caption?

File: _data/navbar.yml (hunk 1 above)
Comment (mheadd) [reply _data/navbar.yml:63]: Good call. How about `Resources`?

File: _projects/ak-medicaid.md (hunk 2 above)
Comment (mheadd) [reply _projects/ak-medicaid.md:7]: There isn't really a connection. I had a hard time finding a picture that was related that was available under a permissive license. Good call on captions though, I'll spin that off into a separate issue.

File: _projects/ca-child-welfare.md (hunk 3 above)
Comment (mheadd) [reply _projects/ca-child-welfare.md:7]: Will create a separate issue for captions for images on example pages.

**Comment by mheadd** [comment]
@awfrancisco I assume the issue with responsive design is related to the template itself. I can't perfectly recreate the issue you describe, but it's still a big improvement over the current version of the site - which [isn't responsive at all](https://github.com/18F/Modular-Contracting-And-Agile-Development/issues/21). Happy to dig into this further though.

**Comment by jposi** [comment]
hi y'all - sorry I'm just catching up on these updates here! I'm trusting
y'all to just keep going for now, excited to review the new work but won't
get to it until tomorrow - I'm fine with y'all pushing to production and
then I'll issue PRs if I see anything that seems wonky to me.

**Review by awfrancisco — APPROVED** [review]

**Review by stvnrlly — APPROVED** [review]

**Comment by randyhart** [comment]
@mheadd The links 404 in the How We're Helping section of https://federalist-proxy.app.cloud.gov/preview/18f/modular-contracting-and-agile-development/site-overhaul/projects/ak-medicaid/

**Comment by mheadd** [comment]
@randyhart Good catch, and apologies that it makes it hard to preview that content. I believe those links 404 because of the way that relative linking works in the preview site on Federalist. Once these changes are deployed to production at the https://modularcontracting.18f.gov/ URL they'll work properly (and they work locally when I test them).

**Comment by jposi** [comment]
this all LGTM.

**Pull request merged**